resource_customizations/kubevirt.io/VirtualMachine/health.lua

```diff
@@ -0,0 +1,18 @@
+hs = { status="Progressing", message="No status available"}
+if obj.status ~= nil then
+  if obj.status.ready ~= nil then
+    hs.status = "Healthy"
+    hs.message = "Running"
+  elseif obj.status.created ~= nil then
+    hs.message = "Starting"
+  end
+  if obj.status.printableStatus ~= nil then
+    hs.message = obj.status.printableStatus
+    if hs.message == "Stopped" or hs.message == "Paused" then
```

Comment:
To minimize reliance on the printableStatus field, we can use the following:
- For "Stopped", you can check that `created == false`.
- For "Paused", you can check the condition with type "Paused" and value "True".